.github/workflows/ci-pr-guidelines.yml

```diff
@@ -0,0 +1,33 @@
+name: ci-pr-guidelines
+
+on:
+  workflow_call:
+    secrets:
+      REPO_GITHUB_TOKEN:
+        required: true
+  pull_request:
+    types:
+      - opened
+    branches-ignore:
+      - 'dependabot/**'
+permissions: 
+  pull-requests: write
+  contents: read
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Let's move this to an example in our README because it's specific to how one might use our new action